building/build-debs/homeworld-keysystem/inner-build.sh

```diff
@@ -26,8 +26,8 @@ export GOPATH="${GODIR}:$(pwd)"
 go build src/keyserver/main/keyserver.go
 go build src/keygateway/main/keygateway.go
 go build src/keyclient/main/keyclient.go
-go build src/keygen.go
-go build src/keyinitadmit.go
-go build src/keyreq.go
+go build src/keygen/main/keygen.go
+go build src/keyinitadmit/keyinitadmit.go
```

Comment:
`main` package source code not in directory called `main`.  I would move this to `src/keyinitadmit/main/keyinitadmit.go`.

Reply:
Try that.